Example/Pods/Local Podspecs/RichTextVC-iOS.podspec.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
I wouldn't commit the Pods in this Example project. They can get them by running Pod install. We do it here, becasue we want to make sure we always have the version of the library we're using. But I wouldn't do it on an open-source project.